xprof/Makefile.am

```diff
@@ -20,8 +20,20 @@ xprof_utils.hpp: $(top_srcdir)/utils/xprof_utils.hpp
 %.pb.h %.pb.cc: %.proto
 	$(PROTOC) -I=$(srcdir) --cpp_out=$(builddir) $<
 
-%/metababel/metababel.h %/metababel/btx_component.h %/metababel/btx_component.c %/metababel/btx_upstream.h %/metababel/btx_upstream.c %/btx_main.c: %_upstream_model.yaml
-	$(METABABEL) -u $< -t SINK -o $* -p $* -c $*
+btx_timeline_upstream_model.yaml btx_tally_upstream_model.yaml &: btx_interval_model.yaml
+	cp $< $(builddir)/btx_timeline_upstream_model.yaml
+	cp $< $(builddir)/btx_tally_upstream_model.yaml
```

Comment:
Split into two commands to avoid using recent group target feature until really required.